runtime/tests/nnapi/src/TestNeuralNetworksWrapper.cpp

```diff
@@ -1,28 +0,0 @@
-/*
- * Copyright (c) 2019 Samsung Electronics Co., Ltd. All Rights Reserved
- * Copyright (C) 2018 The Android Open Source Project
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-#include "TestNeuralNetworksWrapper.h"
-
-namespace nnfw {
-namespace rt {
-namespace test_wrapper {
-
-Execution::ComputeMode Execution::mComputeMode = Execution::ComputeMode::SYNC;
```

Comment:
`TestNeuralNetworksWrapper.cpp` is comes from android source code. Please remain implementation in `runtime/tests/nnapi/src`.

Reply:
OK, I will restore that file.